### DISCUSSION TITLE
Is it possible to use with multiprocessing or joblib?

### DISCUSSION THREAD

**Discussion by jonghwanhyeon** [snakers4/silero-vad#263] [Q&A] [answered]
Hello,

First, I'm really happy to find a good VAD model. Thanks for your contribution.

I'm currently using `multiprocessing` (actually `joblib`) to process a large amount of data samples. I tried to use your model in this `multiprocessing` environment, but it doesn't seem to work.

Is it possible to use your model with multiprocessing? If possible, could you please tell me how to do that?

**Comment by snakers4** [comment, marked as answer]
Hi,

What exactly does not work? 
Which model are you using?

**Reply by jonghwanhyeon** [reply]
Never mind. I solved this issue by using `multiprocessing.Process` rather than `multiprocessing.Pool`.
